UPSTREAM_PROJECTS.yaml

```diff
@@ -195,7 +195,7 @@ projects:
           - commit: 7b286fdc8e8fa91a6e9a179a5494b6ee29fce17b
       - name: hook
         versions:
-          - commit: 6d43b8b331c7a389f3ffeaa388fa9aa98248d7a2
+          - tag: 03a67729d895635fe3c612e4feca3400b9336cc9
```

Comment:
This is a commit right? Confused why it asks for tag here.

@abhay-krishna

Reply:
the build failed when it was commit. 🤷🏼

Reply:
It expects a tag here because of the latest tag pointing to that commit [here](https://github.com/tinkerbell/hook/releases/tag/latest) and the `git ls-remote` check [here](https://github.com/aws/eks-anywhere-build-tooling/blob/main/build/lib/generate_projects_list.sh#L68) dereferences that tag to its commit
```bash
git ls-remote --tags https://github.com/tinkerbell/hook              
65386536b15757bbd132affed3050b0bb261dcab        refs/tags/5.10.57
1b89530156dfa730392b05766da927bd6f565b62        refs/tags/latest
03a67729d895635fe3c612e4feca3400b9336cc9        refs/tags/latest^{}
84e01295b37c15780abbd7db38f408e2df3aba52        refs/tags/v0.6.0
65386536b15757bbd132affed3050b0bb261dcab        refs/tags/v0.6.0^{}
c4d8827191c773d87adf229a48313b576553fa5b        refs/tags/v0.7.0
00dade0c160b13a2ed6e3b980b793f293d043ad4        refs/tags/v0.7.0^{}
```

Can you change that line to
```bash
git ls-remote --tags https://github.com/$org/$repo | awk '{print $2}' | grep ...
```
in the script?

Reply:
thanks @abhay-krishna. Updated. mind checking?

Reply:
It will expect a commit now, so you would need to change it back. Could you run `make generate-project-list` just in case some other project has the same mix-up?

Reply:
Done and pushed. Thanks!

Reply:
Thanks guys!